pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2951,8 +2951,8 @@ int sys_enter_submit(struct bpf_raw_tracepoint_args *ctx)
 
         if (file) {
             fd_arg_task_t fd_arg_task = {
-                .pid = data.context.task.pid,
-                .tid = data.context.task.tid,
+                .pid = data.context.task.host_pid,
```

Comment:
Is this a fix ? If it is, it should be in a single commit as it is unrelated to your change (adding an argument to an event).

Reply:
your'e right. again, not related to this PR.
i should open an issue on this one